File: HLCObjKotlinAct4/src/Java/Act7.java
package Java;

import java.util.Scanner;

public class Act7 {
    public static void main(String[] args) {
        String cadena;
        Scanner sc = new Scanner(System.in);
        System.out.println("Introduce una cadena");
        cadena = sc.nextLine();

        char[] charArray = cadena.toCharArray();
        for(int i = charArray.length - 1; i >= 0; i--) {
            System.out.print(charArray[i]);
        }
    }
}

File: HLCObjKotlinAct4/src/Kotlin/Act7.kt
package Kotlin


fun main() {
    var cadena: String
    println("Introduce una cadena")
    cadena = readln()

    val charArray = cadena.toCharArray()
    println(charArray.indices.reversed())
    for (i in charArray.indices.reversed()) {
        print(charArray[i])
    }
}

